roles/nuage_master/files/serviceaccount.sh

```diff
@@ -59,5 +66,31 @@ oadm create-api-client-config --certificate-authority=${CA_CERT} --client-dir=${
 # Verify the finalized kubeconfig
 if ! [ $(oc whoami --config=$CONFIG_FILE) == 'system:serviceaccounts:default:nuage' ]; then
     echo "Service account creation failed!"
-    exit 1
 fi
+
+
+openssl genrsa -out $REST_CA_KEY 4096
+openssl req -new -x509 -key $REST_CA_KEY -out $REST_CA_CERT -subj "/CN=nuage-signer"
+echo '00' > $REST_CA_SERIAL
+
+OPENSSL_CONFIG='
+[ clientauth ]
+basicConstraints=CA:FALSE
+extendedKeyUsage=critical,clientAuth
+'
+
+echo "$OPENSSL_CONFIG" > ${OUTDIR}/openssl.cnf
+openssl genrsa -out $REST_SERVER_KEY 4096
+openssl req -key $REST_SERVER_KEY -new -out ${OUTDIR}/restServer.req -subj "/CN=$(hostname -f)"
+openssl x509 -req -in ${OUTDIR}/restServer.req -CA $REST_CA_CERT -CAkey \
+    $REST_CA_KEY -CAserial $REST_CA_SERIAL \
+    -out $REST_SERVER_CERT
+
+openssl genrsa -out $REST_CLIENT_KEY 4096
+openssl req -key $REST_CLIENT_KEY -new -out ${OUTDIR}/restClient.req -subj "/CN=nuage-client"
+openssl x509 -req -in ${OUTDIR}/restClient.req -CA $REST_CA_CERT -CAkey \
+    $REST_CA_KEY -CAserial $REST_CA_SERIAL \
+    -out $REST_CLIENT_CERT -extensions clientauth \
+    -extfile ${OUTDIR}/openssl.cnf
```

Comment:
@detiber I am able to generate the certificates for each node on the CA host but I am unable to copy them over. I have pushed my latest changes which exemplify this. 

<pre>
- name: Copy the CA certificate from master to node
  synchronize: src="{{ nuage_ca_crt }}" dest="{{ nuage_plugin_crt_dir }}/nuageMonCA.crt"
  delegate_to: "{{ nuage_ca_master }}" 
</pre>

Obviously this fails because the master node is not setup to do passwordless ssh into the node.
Any idea how do I copy over the certificates ?